docs/bot-optimization.md

```diff
@@ -0,0 +1,157 @@
+the indicators in the file 
+[freqtrade/analyze.py](https://github.com/gcarq/freqtrade/blob/develop/freqtrade/analyze.py#L95-L115).
+Of course you can add more indicators by extending the list contained in
+the function `populate_indicators()`.
+
+Sample:
+```python
+def populate_indicators(dataframe: DataFrame) -> DataFrame:
+    """
+    Adds several different TA indicators to the given DataFrame
+    """
+    dataframe['sar'] = ta.SAR(dataframe)
+    dataframe['adx'] = ta.ADX(dataframe)
+    stoch = ta.STOCHF(dataframe)
+    dataframe['fastd'] = stoch['fastd']
+    dataframe['fastk'] = stoch['fastk']
+    dataframe['blower'] = ta.BBANDS(dataframe, nbdevup=2, nbdevdn=2)['lowerband']
+    dataframe['sma'] = ta.SMA(dataframe, timeperiod=40)
+    dataframe['tema'] = ta.TEMA(dataframe, timeperiod=9)
+    dataframe['mfi'] = ta.MFI(dataframe)
+    dataframe['rsi'] = ta.RSI(dataframe)
+    dataframe['ema5'] = ta.EMA(dataframe, timeperiod=5)
+    dataframe['ema10'] = ta.EMA(dataframe, timeperiod=10)
+    dataframe['ema50'] = ta.EMA(dataframe, timeperiod=50)
+    dataframe['ema100'] = ta.EMA(dataframe, timeperiod=100)
+    dataframe['ao'] = awesome_oscillator(dataframe)
+    macd = ta.MACD(dataframe)
+    dataframe['macd'] = macd['macd']
+    dataframe['macdsignal'] = macd['macdsignal']
+    dataframe['macdhist'] = macd['macdhist']
+    hilbert = ta.HT_SINE(dataframe)
+    dataframe['htsine'] = hilbert['sine']
+    dataframe['htleadsine'] = hilbert['leadsine']
+    dataframe['plus_dm'] = ta.PLUS_DM(dataframe)
+    dataframe['plus_di'] = ta.PLUS_DI(dataframe)
+    dataframe['minus_dm'] = ta.MINUS_DM(dataframe)
+    dataframe['minus_di'] = ta.MINUS_DI(dataframe)
+    return dataframe
+```
+
+## Test your strategy with Backtesting
+Now you have good Buy and Sell strategies, you want to test it against
+real data. This is what we call [backtesting](https://en.wikipedia.org/wiki/Backtesting).
+
+Backtesting will use the crypto-currencies (pair) tickers located in 
+[/freqtrade/tests/testdata](https://github.com/gcarq/freqtrade/tree/develop/freqtrade/tests/testdata).
+If the 5 min and 1 min ticker for the crypto-currencies to test is not 
+already in the `testdata` folder, backtesting will download them 
+automatically. Testdata files will not be updated until your specify it.
+
+### Run a backtesting against the currencies listed in your config file
+**With 5 min tickers (Per default)**
+```bash
+python3 ./freqtrade/main.py -c config.json backtesting --realistic-simulation
+```
+
+**With 1 min tickers**
+```bash
+python3 ./freqtrade/main.py -c config.json backtesting --realistic-simulation --ticker-interval 1
+```
+
+**Reload your testdata files**
+```bash
+python3 ./freqtrade/main.py -c config.json backtesting --realistic-simulation --refresh-pairs-cached
+```
+
+**With live data (do not alter your testdata files)**
+```bash
+python3 ./freqtrade/main.py -c config.json backtesting --realistic-simulation --live
```

Comment:
do we need to pass the `-c` here, shouldn't we use the config by default in backtestging/hyperopt?

Reply:
Removed